### PR TITLE
Kafka rest proxy

### PR DESCRIPTION
**Pull request by Nesze**

### CODE REVIEW

File: service-files/kafka-rest-proxy-sidekick.service [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+[Service]
+ExecStartPre=/bin/sh -c "\
+  etcdctl set /vulcand/backends/kafka-rest-proxy/backend '{\"Type\": \"http\"}' ; \
+  etcdctl set /vulcand/frontends/kafka-rest-proxy/frontend '{\"Type\": \"http\", \"BackendId\": \"kafka-rest-proxy\", \"Route\": \"PathRegexp(`/.*`) && Host(`kafka`)\"}';" 
+
+ExecStart=/bin/sh -c "\
+  export KAFKA_PROXY_IP=$(/usr/bin/ifconfig eth0 | sed -n '2 p' | awk '{print $2}' | cut -d':' -f2); \
+  while true; do \
+    export KAFKA_PROXY_PORT=$(/usr/bin/docker port kafka-rest-proxy 8082 | cut -d':' -f2); \
+    etcdctl set /services/kafka-rest-proxy/1/host '%H' --ttl 600; \
```
Comment (Zapadlo) [inline service-files/kafka-rest-proxy-sidekick.service]: Sorry to be a constant pain about this, but can this path be `/ft/config/...`? Ta!

Comment (Nesze) [inline service-files/kafka-rest-proxy-sidekick.service]: Sure. We are going to change this (on the fly) for all the services, right?

Comment (Zapadlo) [inline service-files/kafka-rest-proxy-sidekick.service]: Trying to deprecate obscure `/services` and put everything into `/ft`, yeah.

**Comment by tosan88** [comment]
+1

**Comment by catalin-ilea** [comment]
+1

**Pull request merged**